Workflow.yml

```
@@ -226,7 +226,8 @@ $graph:
     - name: outputSource
       doc: |
         Specifies one or more workflow parameters that supply the value of to
```

Hmm.. looking back, I think a better wording is needed even from what was in version 1.2.0.
```suggestion
        Specifies one or more names of an output from a workflow step (in the form 
        `step_name/output_name` with a `/` separator`), or a workflow input name,
        that supply their value(s) to the output parameter.
```
And then your other change isn't needed